<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>485</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Быстров М.Д. Z1431</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет стоимости проживания</string>
    </property>
   </widget>
   <widget class="QLabel" name="squareLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь</string>
    </property>
   </widget>
   <widget class="QComboBox" name="squareComboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>15м2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25м2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35м2</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="economRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Эконом</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="businessRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Бизнес</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="luxuryRadioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Люкс</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="foodCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Питание</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="parkingCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Парковочное место</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultTextLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Итого:</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
